### ISSUE TITLE
certificate verification warnings using the INI file for session creation

### ISSUE THREAD

**Issue by entomologe** [Forcepoint/fp-NGFW-SMC-python#41]
Hello,

I'm trying to create a session with an INI file but I'm running into SSL warnings of the following kind:

**/usr/lib/python3/dist-packages/urllib3/connectionpool.py:1015: InsecureRequestWarning: Unverified HTTPS request is being made to host '$IP'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(**

What is the equivalent INI syntax of the following which doesn't throw warnings?

`session.login(url='https://$FQN:8082', api_key='$APIKEY', api_version='6.10', verify=True)
`

How exactly do the above argument "_verify=True_" and the below  "_verify_ssl=True_" differ? According to https://github.com/Forcepoint/fp-NGFW-SMC-python/blob/6b90a325b5405a4250ba24c15852447d6629bbbe/smc/api/configloader.py#L137 the latter verifies client certificates only (and therefore needs a path), but how do I verify the server certificate? 

```
$ cat .smcrc 
smc_address=$IP
smc_apikey=$APIKEY
api_version=6.10
smc_port=8082
smc_ssl=True
verify_ssl=True

```

(c.f. https://fp-ngfw-smc-python.readthedocs.io/en/latest/pages/session.html#creating-the-session )

Thanks.

**Comment by ggrimaux** [comment]
Hello,

I never used uni files but you need to set verify=/path/to/cert.pem like on following example 

from smc import session
session.login(url='https://1.1.1.1:8082', api_key='xxxxxxxxxxxxxxxx',
              verify='/Users/username/home/mycacert.pem')

/Greg.

**Comment by entomologe** [comment]
Hello Greg, thanks for your answer,

I have found the solution to my problem. It turns out it was some rather dumb error on OSI layer 8: I used apostrophes inside the INI file which congruously results in errors I misinterpreted. 

> I never used uni files but you need to set verify=/path/to/cert.pem like on following example

Externalizing credentials resp. having the api_key somewhere else is mandatory in my case.

Thanks again, it helped to narrow it down.
